### PR TITLE
Update eVar64 and add event89

### PR DESCRIPTION
**Pull request by OliverJAsh**
Update loyalty metric from `<days visited>/14` to `<days visited>[/7]`.

event89 is to record this as a count, so it can be filtered by lesser/greater than.

To do:
- [x] Test
- [x] Verify event name
- [x] Check this code is correct with Omniture?

/cc @mkopka @crifmulholland

### CODE REVIEW

File: static/src/javascripts/vendor/omniture.js [hunk 1]
```diff
@@ -69,13 +69,14 @@ function s_doPlugins(s) {
...
         var pV=diary[i];
-        var tmp=(pV>=cutoff) && (pV!=now) && (vC+=1) && newDiary.push(pV);
+        var tmp=(pV>=cutoff) && (pV<now) && (vC+=1) && newDiary.push(pV);
     }
     newDiary.push(now);
     newDiary=newDiary.join(",");
     dtmNow.setFullYear(dtmNow.getFullYear()+5);
     s.c_w("s_daily_habit",newDiary,dtmNow);
-    s.eVar64=vC+"/14";
+    if (s.getVisitStart==1) s.events = s.apl(s.events, 'event16='+vC, ',');
```
Comment (crifmulholland) [inline static/src/javascripts/vendor/omniture.js]: It should actually be event89

**Comment by mkopka** [comment]
Hi can we please make it event89 rather than the current event16

**Comment by OliverJAsh** [comment]
@mkopka That's done.

**Comment by OliverJAsh** [comment]
@mkopka Is this ready to go?

**Comment by mkopka** [comment]
Annoyingly I don't see the event89 also passing the value. Can you?

On Tuesday, 23 June 2015, Oliver Joseph Ash <notifications@github.com>
wrote:

> @mkopka <https://github.com/mkopka> Is this ready to go?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/9568#issuecomment-114416163>.
>


-- 
Sent from Gmail Mobile

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by OliverJAsh** [comment]
It's not deployed yet.

On Wed, 24 Jun 2015 at 08:39 Matt Kopka - Analytics <
notifications@github.com> wrote:

> Annoyingly I don't see the event89 also passing the value. Can you?
>
> On Tuesday, 23 June 2015, Oliver Joseph Ash <notifications@github.com>
> wrote:
>
> > @mkopka <https://github.com/mkopka> Is this ready to go?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/guardian/frontend/pull/9568#issuecomment-114416163>.
> >
>
>
> --
> Sent from Gmail Mobile
>
> --
>
> ------------------------------
> Visit theguardian.com. On your mobile and tablet, download the Guardian
> iPhone and Android apps theguardian.com/guardianapp and our tablet
> editions
> theguardian.com/editions. Save up to 57% by subscribing to the Guardian
> and Observer - choose the papers you want and get full digital access.
> Visit subscribe.theguardian.com
>
> This e-mail and all attachments are confidential and may also be
> privileged. If you are not the named recipient, please notify the sender
> and delete the e-mail and all attachments immediately. Do not disclose the
> contents to another person. You may not use the information for any
> purpose, or store, or copy, it in any way. Guardian News & Media Limited
> is not liable for any computer viruses or other material transmitted with
> or as part of this e-mail. You should employ virus checking software.
>
> Guardian News & Media Limited is a member of Guardian Media Group plc.
> Registered
> Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.
> Registered
> in England Number 908396
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/9568#issuecomment-114766956>.
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by mkopka** [comment]
In dev/ test environment can you see event89 also being sent?

On 24 June 2015 at 10:17, Oliver Joseph Ash <notifications@github.com>
wrote:

> It's not deployed yet.
>
> On Wed, 24 Jun 2015 at 08:39 Matt Kopka - Analytics <
> notifications@github.com> wrote:
>
> > Annoyingly I don't see the event89 also passing the value. Can you?
> >
> > On Tuesday, 23 June 2015, Oliver Joseph Ash <notifications@github.com>
> > wrote:
> >
> > > @mkopka <https://github.com/mkopka> Is this ready to go?
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/guardian/frontend/pull/9568#issuecomment-114416163
> >.
> > >
> >
> >
> > --
> > Sent from Gmail Mobile
> >
> > --
> >
> > ------------------------------
> > Visit theguardian.com. On your mobile and tablet, download the Guardian
> > iPhone and Android apps theguardian.com/guardianapp and our tablet
> > editions
> > theguardian.com/editions. Save up to 57% by subscribing to the Guardian
> > and Observer - choose the papers you want and get full digital access.
> > Visit subscribe.theguardian.com
> >
> > This e-mail and all attachments are confidential and may also be
> > privileged. If you are not the named recipient, please notify the sender
> > and delete the e-mail and all attachments immediately. Do not disclose
> the
> > contents to another person. You may not use the information for any
> > purpose, or store, or copy, it in any way. Guardian News & Media Limited
> > is not liable for any computer viruses or other material transmitted with
> > or as part of this e-mail. You should employ virus checking software.
> >
> > Guardian News & Media Limited is a member of Guardian Media Group plc.
> > Registered
> > Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.
> > Registered
> > in England Number 908396
> >
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/guardian/frontend/pull/9568#issuecomment-114766956>.
> >
>
> --
>
> ------------------------------
> Visit theguardian.com. On your mobile and tablet, download the Guardian
> iPhone and Android apps theguardian.com/guardianapp and our tablet
> editions
> theguardian.com/editions. Save up to 57% by subscribing to the Guardian
> and Observer - choose the papers you want and get full digital access.
> Visit subscribe.theguardian.com
>
> This e-mail and all attachments are confidential and may also be
> privileged. If you are not the named recipient, please notify the sender
> and delete the e-mail and all attachments immediately. Do not disclose the
> contents to another person. You may not use the information for any
> purpose, or store, or copy, it in any way. Guardian News & Media Limited
> is not liable for any computer viruses or other material transmitted with
> or as part of this e-mail. You should employ virus checking software.
>
> Guardian News & Media Limited is a member of Guardian Media Group plc.
> Registered
> Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.
> Registered
> in England Number 908396
>
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/9568#issuecomment-114798872>.
>



-- 



Sent from Gmail Mobile

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Comment by OliverJAsh** [comment]
Tested:

image

image

Look OK? If so I will deploy @mkopka @crifmulholland 

/cc @rich-nguyen I need another :+1:

File: static/src/javascripts/vendor/omniture.js [hunk 2]
```diff
@@ -69,13 +69,14 @@ function s_doPlugins(s) {
     for (var i=0,j=diary.length;i<j;++i)
     {
         var pV=diary[i];
-        var tmp=(pV>=cutoff) && (pV!=now) && (vC+=1) && newDiary.push(pV);
+        var tmp=(pV>=cutoff) && (pV<now) && (vC+=1) && newDiary.push(pV);
```
Comment (rich-nguyen) [inline static/src/javascripts/vendor/omniture.js]: should this really be a `<`?

**Comment by rich-nguyen** [comment]
:+1: , this is exactly how I would implement a client-side analytics system.

File: static/src/javascripts/vendor/omniture.js (hunk 2 above)
Comment (OliverJAsh) [inline static/src/javascripts/vendor/omniture.js]: Yes, apparently.

Comment (crifmulholland) [inline static/src/javascripts/vendor/omniture.js]: Yes, it appears there were issues where we were receiving dates in the future.. (?) so this fixes that.

**Pull request merged**